src/sparseml/transformers/sparsification/obcq/README.md

```diff
@@ -0,0 +1,303 @@
+
+2. Apply to multiple universities: Apply to multiple universities, ensuring that you are applying to the best option for you.
+
+3. Get a job: If you are applying to a university, you will need to find a job to support your studies. This will help you budget and manage your time.
+
+4. Get involved with your community: Your university will likely have a community of students and faculty. Engage with this community by volunteering, participating in clubs, and engaging with others in your community.
+
+5. Get involved with extracurricular activities: Universities often have many extracurricular activities, which can help you meet new people
+"""
+```
+Check out the [DeepSparse pipeline text generation docs](https://github.com/neuralmagic/deepsparse/blob/main/src/deepsparse/transformers/text_generation.md) for the full list of supported parameters. 
+
+## <a name="upload">Upload Model to Hugging Face</a>
+You may want to upload the one-shot model to Hugging Face for ease of reference or to share it with your colleagues. 
+
+Head over to your [Hugging Face account](https://huggingface.co/new) and create a model named `TinyLlama-1.1B-Chat-v0.4-pruned50-quant`. Then upload the one-shot model: 
+```python
+from huggingface_hub import HfApi
+api = HfApi()
+api.upload_folder(
+    folder_path="deployment",
+    repo_id="YOUR_HF_USERNAME/TinyLlama-1.1B-Chat-v0.4-pruned50-quant",
+    repo_type="model",
+    token="HF_WRITE_TOKEN"
+)
+```
+
+## <a name="recipe"> Explaining the TinyLlama Recipe</a>
+A recipe is a set of hyperparameters that provide detailed instructions on how the [one-shot quantization](https://neuralmagic.com/video/pruning-and-quantizing-ml-models-with-one-shot-without-retraining/) should be done. The recipe performs quantization in one shot, meaning that no retraining of the LLM is required. 
+
+We will now walk through what the different hyperparameters mean and why they are set to those values.
+
+The `SmoothQuantModifier` is a technique used for dealing with outliers in the weights and activations of the LLM because quantization is very sensitive to large variations in their values. For TinyLlama a `smoothing_strength` value of 0.8 resulted in a model with repetitions in its output but the problem was solved by lowering the value to 0.5. 
+
+The `ignore` parameter under `QuantizationModifier` allows us to define operations that either don't make sense to quantize or operations that are too sensitive to quantize. Performing quantization on sensitive operations will affect the final accuracy of the model. We also don't quantize the inputs to the embedding layer. 
+
+Under `SparseGPTModifier`, we define `sparsity` as 0.5 because we are aiming for a model that is 50% quantized. The other parameters are:
+- `block_size` determines the number of columns to compress in one pass
+- `quantize` whether or not to quantize weights during SparseGPT
```

Comment:
Maybe worth clarifying that if this is set to "True" and there isn't a QuantizationModifier in the recipe, a default quantization modifier will be applied